### PR TITLE
Easier deployment and bug fix

### PR DESCRIPTION
**Pull request by benel**

### CODE REVIEW

**Comment by benel** [comment]
Cassandre can now work with AAAforREST v2.

Please note that AAAforREST has been fixed ([082a587fbd6a73a1047d06ceb6f88087525b05fe](https://github.com/Hypertopic/AAAforREST/commit/082a587fbd6a73a1047d06ceb6f88087525b05fe)). To update it, first launch (from the Cassandre folder): 

```shell
docker-compose pull
```

**Comment by benel** [comment]
I must add also automatic logging after account creation. Because it is not possible to create the user document if nobody is not logged in. I will force-push the last commit to add this.

**Comment by christophe-lejeune** [comment]
May I ask you to review my last two commits ? Feel free to revert it if you consider it necessary. If you think they are correct, I will merge this pull request. 

Again many thanks for your contribution !

File: docker-compose.yml [hunk 1]
```diff
@@ -27,7 +27,7 @@ services:
     - couchdb
 
   ldap_proxy:
-    image: benel/aaaforrest
+    image: benel/aaaforrest:latest
```
Comment (benel) [inline docker-compose.yml:30]: Isn't it the same?

File: README.md [hunk 2]
```diff
@@ -34,7 +34,7 @@ By default, users are managed both by CouchDB and LDAP.
 
 1. In CouchDB, [create an administrator account](http://localhost:5984/_utils/#createAdmin).
 2. In CouchDB, [create a database](http://localhost:5984/_utils/#/_all_dbs) named `_users`.
-3. Choose a password and set in as the `secret` in `conf/couchdb.ini` and `forwardedSecretLogin`.
+3. Choose a password and set in as the `secret` in `conf/couchdb.ini` and `conf/aaaforrest.yml`.
```
Comment (benel) [inline README.md:37]: It's a good idea to mention the file. 
And it is true that it is called `secret` in both files.
I hope people will understand that `secret` is not `session.secret`.

File: app/l10n/i18n.json [hunk 3]
```diff
@@ -20,7 +20,7 @@
     "i_confirm-password": "Confirm password",
     "i_unmatch-password": "Passwords do not match",
     "i_existing-username": "username already exists",
-    "i_successful-registration": "Your account has been created. Sign-in on the page",
+    "i_successful-registration": "Your account has been created. Start your personal diary or access other's through the page",
```
Comment (benel) [inline app/l10n/i18n.json:23]: Changing the message is definitely a good idea.

File: docker-compose.yml (hunk 1 above)
Comment (christophe-lejeune) [reply docker-compose.yml:30]: According to my experience, the image did not update with `docker-compose pull`. It did after I add `:latest`. 

It also surprised me. However, perhaps it do/should work

**Comment by benel** [comment]
🥂

**Pull request merged**